saleor/graphql/core/types/common.py

```diff
@@ -112,8 +112,8 @@ class GiftCardError(Error):
     code = GiftCardErrorCode(description="The error code.", required=True)
 
 
-class ExtensionsError(Error):
-    code = ExtensionsErrorCode(description="The error code.", required=True)
+class PluginsError(Error):
```

Comment:
This should be singular I think.

Reply:
Other error codes use the singular form so it would be more consistent.